src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/collectionsgrid/view/CollectionGridStyles.kt

```diff
@@ -1,17 +1,22 @@
 package org.wycliffeassociates.otter.jvm.app.ui.collectionsgrid.view
 
 import javafx.geometry.Pos
+import javafx.scene.paint.Color
+import javafx.scene.text.FontWeight
```

Comment:
remove these style imports bc i think all of your styling is in the stylesheets now